akka-typed/src/main/scala/akka/typed/internal/EventStreamImpl.scala

```diff
@@ -190,9 +190,11 @@ private[typed] class EventStreamImpl(private val debug: Boolean)(implicit privat
       }
     } catch {
       case e: Exception ⇒
-        System.err.println("error while starting up loggers")
-        e.printStackTrace()
-        throw new akka.ConfigurationException("Could not start logger due to [" + e.toString + "]")
+        if (!system.whenTerminated.isCompleted) {
+          System.err.println("error while starting up loggers")
+          e.printStackTrace()
+          throw new akka.ConfigurationException("Could not start logger due to [" + e.toString + "]")
+        }
```

Comment:
When the system *is terminated* we swallow the exception.

The system is terminated before the system is fully initalized (here waiting for the logger initialization).
That can happen with Akka typed if the root actor is stopped before the system is fully initalized.

One could perhaps question if the initialization order is correct, i.e. if the root actor should not be created until everything else have been initialized. @rkuhn do you have any thought around that?